go/raindrops: analyzer comments (#1216)
exercism/website-copy: automated-comments/go/raindrops/string_builder_used.md

The `%{function}` is helpful when concatenating a lot of little parts to a string.

> In a use case like this it is overkill. Here the `+=` operator is faster und uses less memory.

Reviewer: "and" not "und"

> In a use case like this it is overkill. Here the `+=` operator is faster and uses less memory.

Reviewer 2: I'm not entirely sure about the word "overkill". It sounds a little bit, well, dramatic. Maybe something like:

```suggestion
For this use case, the `+=` operator is a bit simpler, faster and uses less memory.
```

> This can be verified running the benchmarks as described in the exercise instructions.

Reviewer: "verified by running"
